pkg/storage-init/storage-init.sh

```diff
@@ -155,7 +156,16 @@ for BLK_DEVICE in $BLK_DEVICES; do
 done
 
 #Recording SMART details to a file
-smartctl -a "$(grep -m 1 /persist < /proc/mounts | cut -d ' ' -f 1)" --json > $SMART_DETAILS_FILE
+SMART_JSON=$(smartctl -a "$(grep -m 1 /persist < /proc/mounts | cut -d ' ' -f 1)" --json)
+if [ -f "$SMART_DETAILS_PREVIOUS_FILE" ];
+then
+  rm $SMART_DETAILS_PREVIOUS_FILE 2>/dev/null
```

Comment:
Why do you need this? You can always do the mv which will replace any previous file, then the echo.

Reply:
Removed